polylogue/cli/json_output.py

```
@@ -0,0 +1,143 @@
+
+    Returns:
+        Wrapped handler function
+
+    Example:
+        @safe_json_handler
+        def run_import_cli(args, env):
+            if not file.exists():
+                raise JSONModeError("file_not_found", f"File not found: {file}")
+    """
+
+    def wrapper(args, env):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `wrapper`

(ANN202)